demosplan/DemosPlanCoreBundle/Entity/PlatformFaq.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Entity;
+
+use DateTime;
+use DemosEurope\DemosplanAddon\Contracts\Entities\UuidEntityInterface;
+use demosplan\DemosPlanCoreBundle\Entity\User\Role;
+use demosplan\DemosPlanCoreBundle\Logic\Faq\FaqInterface;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\Entity()
+ */
+class PlatformFaq extends CoreEntity implements UuidEntityInterface, FaqInterface
```

Comment:
I guess "implements UuidEntityInterface," is unnecessary here because FaqInterface already extends the UuidEntityInterface.
Isn't it?